examples/libs/tezos/basic-tx/taquito-injection/.env.example

```diff
@@ -0,0 +1,7 @@
+DFNS_API_URL='https://api.dfns.wtf'
+DFNS_APP_ID='<redacted>'
+DFNS_APP_ORIGIN='http://localhost:3000'
+DFNS_CRED_ID='<redacted>'
+DFNS_PRIVATE_KEY='<redacted>'
+DFNS_AUTH_TOKEN='<redacted>'
+DFNS_WALLET_ID='<redacted>'
```

Comment:
please copy the example format from one of the other examples to demonstrate what the values looks like.

also maybe rename `DFNS_WALLET_ID` to `TEZOS_WALLET_ID`.